src/styles/blocks/catalog.scss

```
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $smooth, color $smooth;
+
+    &:hover {
+      color: #34568b;
```

You create a variable for teal color, but here color also wants to be a variable 🥺 
```suggestion
      color: $c-color;
```